ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -46,4 +67,14 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
     func searchBarDidRequestSearch(_ searchBar: SearchBar) {
         searchBar.progressSpinner.state.isAnimating = true
     }
+
+    func searchBar(_ searchBar: SearchBar, didUpdateBadgeView badgeView: BadgeView?) {
```

Comment:
This implementation hasn't been updated to match the changes in the protocol that you have made.

Reply:
Fixed